From d41f92047a8f2c1e1f31ff8939b2215862bce183 Mon Sep 17 00:00:00 2001
From: Fedora GDB patches <invalid@email.com>
Date: Fri, 27 Oct 2017 21:07:50 +0200
Subject: gdb-readline62-ask-more-rh.patch

FileName: gdb-readline62-ask-more-rh.patch

;; Work around readline-6.2 incompatibility not asking for --more-- (BZ 701131).
;;=fedora
---
 gdb/event-top.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/gdb/event-top.c b/gdb/event-top.c
index a1866f0c88..e7d3c1fc20 100644
--- a/gdb/event-top.c
+++ b/gdb/event-top.c
@@ -1182,6 +1182,13 @@ gdb_setup_readline (int editing)
 {
   struct ui *ui = current_ui;
 
+#ifdef NEED_RL_STATE_FEDORA_GDB
+  /* 6.2 regression: no longed asks for --more--
+     gdb.base/readline-ask.exp
+     https://bugzilla.redhat.com/show_bug.cgi?id=701131  */
+  RL_SETSTATE (RL_STATE_FEDORA_GDB);
+#endif
+
   /* This function is a noop for the sync case.  The assumption is
      that the sync setup is ALL done in gdb_init, and we would only
      mess it up here.  The sync stuff should really go away over
-- 
2.14.3

